src/dms/core/EdFi.DataManagementService.Core.External/Backend/UpdateCascadeResult.cs

```diff
@@ -34,11 +35,11 @@ public record UpdateCascadeResult(
     /// <summary>
     /// The project name of the referencing resource
     /// </summary>
-    string ProjectName,
+    ProjectName ProjectName,
     /// <summary>
     /// The name of the referencing resource
     /// </summary>
-    string ResourceName,
+    ResourceName ResourceName,
```

Comment:
Type branding was helpful in identifying a subtle bug where endpoint names and resource names were assumed equivalent but are not.